### ISSUE TITLE
Static page wireframes 

### ISSUE THREAD

**Issue by elainekamlley** [LibraryOfCongress/concordia#273]
**Is your feature request related to a problem? Please describe.**
Our current static pages (about & instructions) don't have much styling. 

**Describe the solution you'd like**
These are internal pages that provide our users with contextual information to perform tasks on the application. 

- [x] A wireframe for our `/about` page that helps our users read through text and helps them understand our purpose and mission. 
- [x] A wireframe for our `/instructions` page that takes users through the process step by step. We possibly want to use `.gif`s to capture the process

**Additional context**

![oldstatic](https://user-images.githubusercontent.com/7362915/46026905-78ec4400-c0ba-11e8-9f46-f8c9dc467cb1.PNG)

link to about content: [link here]

**Comment by jbresner** [comment]
Here's a first pass at an About page that includes the FAQs. The FAQs are initially all collapsed and I'm thinking initially only one can be expanded at a time (i.e. if one's already opened, and user clicks on second one, the first one collapses). I'll try to get more detailed specs in place before close of sprint.

![about-v1](https://user-images.githubusercontent.com/4275957/46385376-f0593f00-c689-11e8-8cf2-a2569df7aa8a.png)
![about-v2](https://user-images.githubusercontent.com/4275957/46385378-f51df300-c689-11e8-8475-f28d904ca9a3.png)
![about-v2-expanded](https://user-images.githubusercontent.com/4275957/46385381-f7804d00-c689-11e8-9b3d-fd19bf6d5a77.png)

**Comment by jbresner** [comment]
![instructions-short-v1](https://user-images.githubusercontent.com/4275957/46386511-d7538c80-c68f-11e8-9d97-5028efc7798a.png)

Here's a first pass at the "Short" set of instructional copy.

A couple thoughts:
1. We need to establish the business rules as Victoria mentioned above (i.e. where and when does this appear and if it's dismissed, is a user ever able to see this again?). For an unauthenticated user, I like the idea of this showing up for the first time only after a user clicks on any "Get Started" button. I'm not sure I'd ever display this for signed-in users but let's figure out the other instructional pages/modal first before finalizing this decision.
2. The modal may be dismissed by clicking/tapping the "X" or clicking anywhere outside of the white box.
3. Clicking/tapping the left/right arrows will move to the next/previous slide. Clicking/tapping the circle indicators will also allow the user to jump to that corresponding slide. The "off" state of the indicator is the white version. The "on" state of the indicator is the orange version. 
4. The previous button should be disabled when viewing the first slide. The next button should be disabled when viewing the last slide.
5. The colored box will be replaced with an appropriate screen shot (this could also be an animated GIF if it helps better convey the information).

**Comment by jbresner** [comment]
Here are the final comps. I have two proposed approaches. The first one is similar to what was previously built (the sliding tray displays the instructional copy). The second version is a different approach that instead adds a link to the global nav and combines the FAQs with links to articles (e.g. How to Review). 

While I understand the desire to have the instructional copy as part of the UI as users are trying to use the site, I'm not sure how beneficial it is once they understand what it is they're doing and it then gets in the way of the UI.

The latter approach uses a centralized "Help Center" that combines the articles (these can grow and be categorized as needed), the FAQs, an inline "Contact" form, and a link to the Onboarding modals (i.e. the Short Instructions). I understand a lot of this may be aspirational, but I still see some benefits to keeping the Help section separate from the primary functionality of the application.

Let me know your thoughts.

![transcribe-anon](https://user-images.githubusercontent.com/4275957/46430333-b124ff80-c716-11e8-9e14-ea820cb58f86.png)
![help tray](https://user-images.githubusercontent.com/4275957/46430343-b5e9b380-c716-11e8-9389-f949ad639e13.png)
![transcribe-anon-helpnav](https://user-images.githubusercontent.com/4275957/46430350-b8e4a400-c716-11e8-97b0-564ef0f6d255.png)
![help center](https://user-images.githubusercontent.com/4275957/46430354-bb46fe00-c716-11e8-8d40-da3739305646.png)
![help center article](https://user-images.githubusercontent.com/4275957/46430356-be41ee80-c716-11e8-8c9b-31cd8d9506b5.png)

**Comment by jbresner** [comment]
![transcribe-anon-nothingtotranscribe](https://user-images.githubusercontent.com/4275957/46495193-bf405200-c7e2-11e8-87db-14f888cc1f2b.png)
Made some quick changes based on conversation.

**Comment by jbresner** [comment]
I'm attaching three updated mockups.

The first is a revised version of the "Help Center". This page still has two main sections: a "Instructions" section and the "FAQ" section that opens each answer via an accordion UI. The "Instructions" section has been updated to include a link to the on-boarding content a user would first see after clicking "Get Started" for the first time. Instead of displaying as a modal (as it would for the first time viewers), I've added the content as a detail page to be consistent with the other instruction pages.

The other two mockups are the "Get Started/Welcome" page and the previously provided "How to Review" page.

![help center](https://user-images.githubusercontent.com/4275957/46691399-dca26100-cbd2-11e8-9320-f4445b6858d8.png)
![help center article - welcome](https://user-images.githubusercontent.com/4275957/46691403-de6c2480-cbd2-11e8-9f01-c53e38e23cd0.png)
![help center article](https://user-images.githubusercontent.com/4275957/46691408-e035e800-cbd2-11e8-94e7-c8bf95507c52.png)
